app/controllers/application_controller.rb

```diff
@@ -18,4 +19,9 @@ def authenticate_user!
       redirect_to new_user_session_path, :notice => 'please sign in'
     end
   end
+
+def get_emergencies_list
+    @emergencies_list = Emergency.last(5).reverse
```

Comment:
last by what? It should be by `updated_at`. So it should be `order(updated_at: :desc).limit(5)`